src/net/tcp.ts

```diff
@@ -475,7 +473,7 @@ export class Connection {
   /**
    * Sends a message that is pre-formatted as an arrayBuffer.
    */
-  public send = (msg :ArrayBuffer) : Promise<freedom_TcpSocket.WriteInfo> => {
+  public send = (msg :ArrayBuffer) : Promise<void> => {
```

Comment:
I added a comment here noting that the Promise can reject under certain circumstances.  I think that might make the return worthwhile.